components/ws-manager-mk2/controllers/suite_test.go

```diff
@@ -99,6 +99,8 @@ var _ = BeforeSuite(func() {
 				Name: "default",
 			},
 		},
+		WorkspaceURLTemplate: "{{ .ID }}-{{ .Prefix }}-{{ .Host }}",
+		GitpodHostURL:        "gitpod.io",
```

Comment:
the value of this doesn't really matter at the moment, it needs to be non-empty so that a workspace URL can be computed and gets set in the workspace status (which the test then verifies that it gets set by the controller)

Reply:
Okay, gotcha! For the future, let's consider using fake values, like `foo.bar`, if it's not really meaningful. Marking as resolved for now.